docs/DeveloperGuide.adoc

```diff
@@ -122,21 +122,22 @@ image::ModelClassDiagram.png[]
 The `Model`,
 
 * stores a `UserPref` object that represents the user's preferences.
-* stores the Address Book data.
-* exposes an unmodifiable `ObservableList<Person>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
+* stores the Library data.
+* stores a `GameManager` object that represents one game session.
+* stores a `Deck` object that represents the deck that the user is viewing when user is in deck view.
+* stores a `Card` object that represents the card that the user is playing with when user is in play mode.
+* stores a `View` object that represents the view that the user is currently in.
+* stores a `Mode` object that represents the mode that the user is currently in.
+* exposes an unmodifiable `ObservableList<Deck>` that can be 'observed' e.g. the UI can be bound to this list so that the UI automatically updates when the data in the list change.
 * does not depend on any of the other three components.
```

Comment:
In Figure 7, I think you can improve on it by showing the multiplicities of the Front Face, Back Face of each card and also between Deck and Name, etc.